### ISSUE TITLE
Discard bad performance data

### ISSUE THREAD

**Issue by shinglyu** [shinglyu/servo-perf#3]
Got this from `performance.timing`
``` 
 {
  ¦ "unloadEventStart": null, 
  ¦ "domLoading": 1462328699275, 
  ¦ "fetchStart": null, 
  ¦ "responseStart": null, 
  ¦ "loadEventEnd": 0,  
  ¦ "connectStart": null, 
  ¦ "domainLookupStart": null, 
  ¦ "redirectStart": null, 
  ¦ "domContentLoadedEventEnd": 1462328701612, 
  ¦ "requestStart": null, 
  ¦ "secureConnectionStart": null, 
  ¦ "connectEnd": null, 
  ¦ "navigationStart": 1462328699274, 
  ¦ "loadEventStart": 0,  
  ¦ "domInteractive": 1462328701273, 
  ¦ "domContentLoadedEventStart": 1462328701612, 
  ¦ "redirectEnd": null, 
  ¦ "domainLookupEnd": null, 
  ¦ "unloadEventEnd": null, 
  ¦ "responseEnd": null, 
  ¦ "testcase": "http://localhost:8000/page_load_test/aljazeera.net/aljazeera.net/portal.html", 
  ¦ "domComplete": 0
  },  
```
domComplete = 0 => domComplete - navigationStart < 0 => Geometric mean breaks. Need to drop this measurement.

**Comment by jdm** [comment]
We could implement domComplete, too :p

**Comment by shinglyu** [comment]
@jdm: I thought domComplete was implemented? Only one of the test gives me 0, others are correct.

**Comment by jdm** [comment]
Whoops, you're right!

**Comment by shinglyu** [comment]
Probably because I kill the test after 1 min timeout. I should make the timeout more generous and also filter out these timed-out tests in the code.

**Comment by shinglyu** [comment]
Closed by d22a2a899ac273d5e9ad0f259494d9bd210c164d
